src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1166,6 +1166,8 @@ internal virtual void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleB
         /// </summary>
         public abstract bool AreLocalsZeroed { get; }
 
+        internal virtual bool IsNullableEnabled() => false;
```

Comment:
We generally try not to put virtual members on the top-level symbol types, and prefer to implement them in the derived types to make sure that everything is forwarded correctly. Consider doing that here.

Reply:
This doesn't appear to be addressed.

---
In reply to: [546856989](https://github.com/dotnet/roslyn/pull/49953#discussion_r546856989) [](ancestors = 546856989)